posthog/api/feature_flag.py

```diff
@@ -68,6 +69,29 @@ def validate_key(self, value):
 
         return value
 
+    def validate_filters(self, filters):
+        aggregation_group_type_index = filters.get("aggregation_group_type_index", None)
+
+        def properties_all_match(predicate):
+            return all(
+                predicate(Property(**property))
+                for condition in filters["groups"]
+                for property in condition.get("properties", [])
+            )
+
+        if aggregation_group_type_index is None:
+            is_valid = properties_all_match(lambda prop: prop.type in ["person", "cohort"])
+            if not is_valid:
+                raise serializers.ValidationError("Filters are not valid (can only use person and cohort properties)")
+        elif aggregation_group_type_index is not None:
```

Comment:
non-blocking

I guess this is equivalent to just else? To communicate that this is an either or to future developers perhaps just use `else:`?

Reply:
You're right, it should be `else`. Victim of refactoring :)